src/olympia/abuse/cinder.py

```diff
@@ -98,6 +92,28 @@ def get_context(self):
         }
 
 
+class CinderNonFxaUser(Cinder):
```

Comment:
> anonymous and logged-in users 

These users aren't anonymous in the reporting sense, but aren't logged-in users either.  The naming was because we won't be sending the `fxa_id` but is `CinderUnauthenticatedReporter` (like the type string we'd use) better?